crypto/eth2_keystore/src/json_keystore/kdf_module.rs

```diff
@@ -4,6 +4,7 @@
 //! data structures. Specifically, there should not be any actual crypto logic in this file.
```

Comment:
I think we're defying this comment by adding the validation here. We could either remove the comment or move these checks into `keystore.rs`. I prefer the latter, personally.

Reply:
Ooh yep I didn't see that I'll move the functionality to `keystore.rs`